dipy/sims/voxel.py

```diff
@@ -16,6 +16,22 @@
 diffusion_evals = np.array([1500e-6, 400e-6, 400e-6])
 
 
+def _check_directions(angles):
+    """
+    Helper function to check if direction ground truth have the right format
+    and are in cartesian coordinates
```

Comment:
Even though it's an internal function, it still needs a proper docstring (Parameters, Returns)